app/lib/submission_builder/shared/return_header1040.rb

```diff
@@ -73,7 +73,7 @@ def document
               xml.USAddress {
                 # IRS provides no information on how to shorten addresses
                 # but requires that they be < 36 characters long
-                xml.AddressLine1Txt trim([address.urbanization, address.street_address].map(&:presence).compact.join(", "), 35)
+                xml.AddressLine1Txt trim([address.urbanization, address.street_address].map(&:presence).compact.join(" "), 35)
```

Comment:
i feel unsure why we had to do a presence.compact here, but i remember testing it in the console and realizing it was in fact necessary

Reply:
`""` vs `nil`

Travis and I worked somewhat hard to avoid storing a `""` in the database for `.urbanization`, trying to instead either store `nil` or an actual value, but we might have gotten it wrong, and no point in extra space characters showing up.